.github/scripts/generate_ci_workflows.py

```diff
@@ -71,7 +131,7 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
         with open(output_file_path, "w") as output_file:
             GENERATED = "generated"
             output_file.writelines([f"# @{GENERATED} DO NOT EDIT MANUALLY\n"])
-            output_file.write(workflow_template.render(**asdict(workflow)))
+            output_file.write(workflow_template.render(asdict(workflow)))
```

Comment:
I don't quite understand this change; could you explain?

Reply:
it also takes a dict object, so saves some time and no need to convert to `**kwargs`